src/utils/showToasts.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @author Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {
+	showError as baseShowError,
+	showInfo as baseShowInfo,
+	showMessage as baseShowMessage,
+	showSuccess as baseShowSuccess,
+	showUndo as baseShowUndo,
+	showWarning as baseShowWarning,
+	TOAST_UNDO_TIMEOUT,
+	TOAST_DEFAULT_TIMEOUT,
+	TOAST_PERMANENT_TIMEOUT,
+	TOAST_ARIA_LIVE_OFF,
+	TOAST_ARIA_LIVE_POLITE,
+	TOAST_ARIA_LIVE_ASSERTIVE,
+} from '@nextcloud/dialogs'
+
+import store from '../store/index.js'
+
+/**
+ * Add a reference to Talk app container ID if no other selectors are provided
+ * This is necessary to render toast messages in Fullscreen mode properly
+ *
+ * @param {Function} baseFunction callback of 'nextcloud/dialogs'
+ */
```

Comment:
**issue (blocking):** JSDoc is not correct:
* `baseFunction` is not a callback
* No `@return`
* We may set less base type to note lose auto-complete in IDE


**suggestion:** add return and update `baseFunction description

```js
/**
 * Add a reference to Talk app container ID if no other selectors are provided
 * This is necessary to render toast messages in Fullscreen mode properly
 *
 * @param {function} baseFunction showX function from '@nextcloud/dialogs'
 * @return {function} wrapped baseFunction with a new default selector
 */
```

**suggestion:** using `@template` (not a part of the standard but works). It adds auto-complete for new functions.

```js
/**
 * Add a reference to Talk app container ID if no other selectors are provided
 * This is necessary to render toast messages in Fullscreen mode properly
 *
 * @template {Function} T
 * @param {T} baseFunction showX function from '@nextcloud/dialogs'
 * @return {T} wrapped function with a new default selector
 */
```

More strict typing require `@nextcloud/dialogs` improvements, because is doesn't export types.